lib/internal/util.js

```diff
@@ -114,8 +114,7 @@ exports.normalizeEncoding = function normalizeEncoding(enc) {
       case 'hex':
         return enc;
       default:
-        if (low)
-          return undefined;
+        if (low) return; // undefined
```

Comment:
If we're going to add the comment we may as well just use `return undefined;` ;-)